test/Microsoft.ML.TestFrameworkCommon/Attributes/EnvironmentSpecificTheoryAttribute.cs

```diff
@@ -5,7 +5,7 @@
 using System;
 using Xunit;
 
-namespace Microsoft.ML.TestFramework.Attributes
+namespace Microsoft.ML.TestFrameworkCommon.Attributes
```

Comment:
Do we need to use `TestFrameworkCommon` in the namespace? I think it would cut down on a lot of churn by just using `Microsoft.ML.TestFramework.Attributes` even though these attributes are supplied by the `TestFrameworkCommon` assembly.  Note that assembly and namespace do not need to match.

We only have TestFrameworkCommon so we don't have to reference the full TestFramework library in all places.

Reply:
Honestly I had forgotten that the namespace and assembly didn't have to match. We don't have to use the `TestFrameworkCommon` if I change the namespace for those attributes. I'll go ahead and fix that.

Reply:
So looking at the code again, the reason that I made this change is that `EnvironmentSpecificFactAttribute` was in `TestFrameworkCommon` and it seemed inconsistent to have the Fact be in a different namespace then the Theory. 

I think we should leave them the same, but I am happy to change them both to not be in common. Thoughts?

Reply:
I'd be fine with putting them all in `Microsoft.ML.TestFramework.Attributes`

Reply:
Alright, they have been moved to that namespace.

Reply:
I'm still seeing attributes in the `TestFrameworkCommon` namespace.

Reply:
this still isn't resolved.

Reply:
Just checked and this is the only attribute that isn't resolved. The ones I created are in TestFramework, and the others have been left as is. I'll resolve this now.